dist/angular-selector.js

```diff
@@ -27,6 +27,7 @@
 				labelAttr:             '@?',
 				groupAttr:             '@?',
 				options:               '=?',
+				create:                '=?',
```

Comment:
This should be `"&?"`, otherwise if you use a custom "create" function your are not able to see contents where `ng-if="create"` is set.